caiman/motion_correction.py

```diff
@@ -768,7 +769,7 @@ def motion_correct_online(movie_iterable, add_to_movie, max_shift_w=25, max_shif
                 big_mov[:, idx_frame] = np.reshape(
                     new_img, np.prod(dims[1:]), order='F')
 
-            if return_mov and (n_iter == (n + 1)):
+            if mov is not None and (n_iter == (n + 1)):
```

Comment:
This should not change functionality, and makes it clear to the type inference engine that mov has a value for this block